### PR TITLE
fixed json formate for Canvas -lms

### PR DESCRIPTION
**Pull request by Aditi8007**
https://github.com/Be-Secure/Be-Secure/issues/189 fixed json formate

### CODE REVIEW

**Review by sudhirverma — COMMENTED** [review]
LGTM can you please add criticality_score too.

File: canvas-android/teacher-1.19.0-52/criticality_score/canvas-android-teacher-1.19.0-52-criticality -score.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (sudhirverma) [inline canvas-android/teacher-1.19.0-52/criticality_score/canvas-android-teacher-1.19.0-52-criticality -score.json:1]: @Aditi8007 can you check file name is correct I see space after criticality

canvas-android-teacher-1.19.0-52-criticality -score.json

File: canvas-lms/release-2023-03-15.23/criticality_score/canvas-lms-release-2023-03-15.23-criticality -score.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (sudhirverma) [inline canvas-lms/release-2023-03-15.23/criticality_score/canvas-lms-release-2023-03-15.23-criticality -score.json:1]: same here

File: kaa/1.4.13/criticality_score/kaa-1.4.13-criticality -score.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
+    "name": "kaa",
```
Comment (sudhirverma) [inline kaa/1.4.13/criticality_score/kaa-1.4.13-criticality -score.json:2]: same here

File: pimcore/10.1.0/criticality_score/pimcore-v10.1.0-criticality -score.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
+{
+    "name": "pimcore",
+    "url": "https://github.com/Be-Secure/pimcore",
```
Comment (sudhirverma) [inline pimcore/10.1.0/criticality_score/pimcore-v10.1.0-criticality -score.json:3]: same here

File: ros/1.15.7/criticality_score/ros-1.15.7-criticality -score.json [hunk 5]
```diff
@@ -0,0 +1,18 @@
+{
+    "name": "ros",
+    "url": "https://github.com/Be-Secure/ros",
```
Comment (sudhirverma) [inline ros/1.15.7/criticality_score/ros-1.15.7-criticality -score.json:3]: same here

**Review by sudhirverma — CHANGES_REQUESTED** [review]

File: canvas-android/teacher-1.19.0-52/criticality_score/canvas-android-teacher-1.19.0-52-criticality-score.json [hunk 6]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (sudhirverma) [inline canvas-android/teacher-1.19.0-52/criticality_score/canvas-android-teacher-1.19.0-52-criticality-score.json:1]: You are creating the duplicate report

**Review by sudhirverma — CHANGES_REQUESTED** [review]

**Review by asa1997 — APPROVED** [review]
@Aditi8007 @harimohanr LGTM

**Pull request merged**